### DISCUSSION TITLE
Deprecation Warning From Node

### DISCUSSION THREAD

**Discussion by susansiow** [quasarframework/quasar#13332] [General - Components / Directives / etc] [answered]
I have two Node versions installed locally - 14.15.5 and 16.14.2. When I apply the later one (16.14.2) and run `quasar info` command, I will receive a depreciation warning from Node about Vuex:

```
(node:14512) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at [path]\node_modules\vuex\package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
```

Quasar package versions:

```
// Global
@quasar/cli - 1.3.2

// Dependencies
quasar - 1.18.10
@quasar/extras - 1.13.6

// Dev Dependencies
@quasar/app - 2.3.0
```
Nevertheless, Vuex seems to be working smoothly (despite the warning). Is this a false alarm and can be ignored?

**Comment by Hashen110** [comment, marked as answer]
Upgrade vuex version to [v3.6.2](https://github.com/vuejs/vuex/releases/tag/v3.6.2)

**Reply by susansiow** [reply]
<img width="257" alt="screenshot" src="https://user-images.githubusercontent.com/53169547/166865205-e08792ef-de0b-4400-95b7-8ce2421a2813.png">

The Vuex is initially installed via Quasar CLI v1 and the Vuex version is `v3.6.2` (as specified in node_modules/vuex/package.json). Secondly, when I run `quasar info` command, the error persists with the application of Node v16.14.2.

```
(node:14512) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at [path]\node_modules\vuex\package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
```

```
// Global
@quasar/cli - 1.3.2

// Dependencies
quasar - 1.18.10
@quasar/extras - 1.13.6

// Dev Dependencies
@quasar/app - 2.3.0
```

Kindly assist further, thank you!

**Reply by metalsadman** [reply]
see https://github.com/quasarframework/quasar/discussions/13335

**Reply by susansiow** [reply]
Checked the [solution](https://github.com/quasarframework/quasar/discussions/13335#discussioncomment-2685454), Vuex is working though the error message, so I can in fact take no. 4 > 'ignore warning'?

**Reply by MilosPaunovic** [reply]
Yes, you can ignore it.

**Reply by susansiow** [reply]
Many thanks!
